android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/logto/LogtoWebViewAuthClient.kt

```diff
@@ -1,28 +1,28 @@
-package io.logto.sdk.android.auth.webview
+package io.logto.sdk.android.auth.logto
 
 import android.app.Activity
-import android.content.Intent
+import android.graphics.Bitmap
 import android.webkit.WebResourceRequest
 import android.webkit.WebView
 import android.webkit.WebViewClient
-import io.logto.sdk.android.auth.LogtoAuthManager
 
-class WebViewAuthClient(
+class LogtoWebViewAuthClient(
     private val hostActivity: Activity,
+    private val injectScript: String,
 ) : WebViewClient() {
+    override fun onPageStarted(view: WebView?, url: String?, favicon: Bitmap?) {
+        super.onPageStarted(view, url, favicon)
+        view?.evaluateJavascript(injectScript, null)
```

Comment:
What JavaScript code do we inject into the webview?

Reply:
https://github.com/logto-io/kotlin/pull/132/files#diff-be0402f1c7797833d11f1933f48fdcb18b2c19f8a6fb94b05664f8d2b92d1c14R26-R30